cli/src/commands/developer/query.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019-2025 Provable Inc.
+// This file is part of the snarkOS library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod transaction;
+pub use transaction::QueryTransaction;
+
+use super::DEFAULT_ENDPOINT;
+use crate::helpers::args::prepare_endpoint;
+
+use snarkvm::prelude::Network;
+
+use anyhow::Result;
+use clap::Parser;
+use ureq::http::Uri;
+
+#[derive(Debug, Parser)]
+pub enum QueryCommand {
+    Transaction(QueryTransaction),
+}
+
+#[derive(Debug, Parser)]
+pub struct Query {
```

Comment:
We should remove this for now. In the meantime `leo` has all this functionality already in their CLI. 

In the future, we can adjust snarkVM `Query` or some base level repo (`aleo-std`) to define all the query endpoints for use in the higher level reps like `snarkVM`, `snarkOS`, and `leo`.